### DISCUSSION TITLE
Potential permissions bypass in Apiman 1.5.7 through Apiman 2.2.3.Final (CVE-2022-47551)

### DISCUSSION THREAD

**Discussion by msavy** [apiman/apiman#2409] [Announcements]
CVE  in Apiman Manager REST API Apiman 1.5.7 through Apiman 2.2.3.Final has been assigned **CVE-2022-47551**

Read more about this Apiman CVE: https://www.apiman.io/blog/permissions-bypass-disclosure/

**Comment by alograg** [comment]
Hi everyone
Where can I find an example of this CVE to help build a patch for version 2.2.x?
Thanks
